doozerlib/cli/__main__.py

```diff
@@ -1518,6 +1522,132 @@ def config_read_group(runtime, key, as_len, as_yaml, permit_missing_group, defau
     print(str(value))
 
 
+def get_releases(group) -> dict:
+    """
+    Uses GitHub API to fecth releases.yaml from openshift-eng/ocp-build-data for a given group
+
+    Parses the file and returns it as a dictionary
+    """
+
+    if not os.environ.get('GITHUB_TOKEN', None):
+        raise DoozerFatalError('A GITHUB_TOKEN environment variable must be defined!')
+    github_token = os.environ['GITHUB_TOKEN']
+
+    api = GhApi(owner='openshift-eng', repo='ocp-build-data', token=github_token)
```

Comment:
That might cause a problem if people use their ocp-build-data fork.
I think converting ocp-build-data url to (owner, repo) is needed.

Reply:
Fixed in 14d095339559c1693158458ba2a1c706632a4488